src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Utils/SPDXToSbomFormatConverterExtensions.cs

```diff
@@ -15,36 +18,186 @@ namespace Microsoft.Sbom.Utils;
 /// </summary>
 public static class SPDXToSbomFormatConverterExtensions
 {
-    /// <summary>
-    /// Converts a <see cref="SPDXFile"/> object to a <see cref="SbomFile"/> object.
-    /// </summary>
-    /// <param name="spdxFile"></param>
-    /// <returns></returns>
-    public static SbomFile ToSbomFile(this File spdxFile)
+    public static SbomFile ToSbomFile(this File spdxFile, List<Element> spdx30Elements = null, List<Relationship> relationships = null)
     {
-        var checksums = spdxFile.VerifiedUsing?.Select(c => c.ToSbomChecksum());
-
-        // Not setting LicenseConcluded and LicenseInfoInFiles since the whole SBOM is required to set these values.
         return new SbomFile
         {
-            Checksum = checksums,
-            FileCopyrightText = spdxFile.CopyrightText,
+            Checksum = spdxFile.VerifiedUsing?.ToSbomChecksum(),
+            FileCopyrightText = spdxFile.CopyrightText == "NOASSERTION" ? null : spdxFile.CopyrightText,
+            Path = spdxFile.Name,
             Id = spdxFile.SpdxId,
-            Path = spdxFile.Name
+            LicenseConcluded = spdx30Elements != null && relationships != null
+            ? spdxFile.GetSingleLicense(RelationshipType.HAS_CONCLUDED_LICENSE, spdx30Elements, relationships)
+            : null,
+            LicenseInfoInFiles = spdx30Elements != null && relationships != null
+            ? spdxFile.GetMultipleLicenses(RelationshipType.HAS_DECLARED_LICENSE, spdx30Elements, relationships)
+            : null
+        };
+    }
+
+    public static SbomPackage ToSbomPackage(this Package spdxPackage, List<Element> spdx30Elements, List<Relationship> relationships)
+    {
+        var sbomPackage = new SbomPackage
+        {
+            PackageName = spdxPackage.Name,
+            PackageVersion = spdxPackage.PackageVersion,
+            PackageSource = spdxPackage.DownloadLocation == "NOASSERTION" ? null : spdxPackage.DownloadLocation,
```

Comment:
nit: we use the string "NOASSERTION" a lot throughout this class, can we keep it as a const instead of duplicating the string every time?